tests/test_storageclass_invalid.py

```diff
@@ -0,0 +1,68 @@
+import logging
```

Comment:
I suggest aggregating a few test cases, that have the same product functionality/allow code reuse, inside this test file.